docs/WEATHER.md

```
@@ -0,0 +1,200 @@
+```env
+CDSAPI_KEY=your-cds-api-key-here
+```
+
+Get your key at: https://cds.climate.copernicus.eu/
+
+## Output Format
+
+Data is saved with date ranges in filename:
+`data/{source}/processed/{country}_{source}_weather_{start_date}_to_{end_date}.csv`
+
+### CSV Structure
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)